kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ConsumerGroupService.java

```diff
@@ -209,6 +209,27 @@ private Mono<List<ConsumerGroupDescription>> describeConsumerGroups(ReactiveAdmi
         .map(cgs -> new ArrayList<>(cgs.values()));
   }
 
+
+  private Mono<List<ConsumerGroupDescription>> loadDescriptionsByInternalConsumerGroups(ReactiveAdminClient ac,
+                                                                                  List<ConsumerGroupListing> groups,
+                                                                                  Comparator<GroupWithDescr> comparator,
+                                                                                  int pageNum,
+                                                                                  int perPage,
+                                                                                  SortOrderDTO sortOrderDto) {
+    var groupNames = groups.stream().map(ConsumerGroupListing::groupId).toList();
+
+    return ac.describeConsumerGroups(groupNames)
+    .flatMap(descriptionsMap -> {
```

Comment:
please format code using `/etc/checkstyle.xml`

Reply:
The build finishes successfully without any errors or warnings (from the checkstyle plugin) regarding style issues here. 
What exactly is the issue ? what should I do ?